src/app.js

```diff
@@ -0,0 +1,124 @@
+const headline = (id, title, webUrl, apiUrl) => {
+  return {
+    id,
+    title,
+    webUrl,
+    apiUrl
+  }
+};
+
+class App {
+  constructor(makeHeadlinesRequest, makeSummaryRequest, makeArticleRequest) {
+    this.makeHeadlinesRequest = makeHeadlinesRequest;
+    this.makeSummaryRequest = makeSummaryRequest;
+    this.makeArticleRequest = makeArticleRequest;
+  }
+
+  getHeadlines () {
+    this.makeHeadlinesRequest(data => {
+      let idx = 0;
+      data.response.results.forEach((item) => {
+        idx++;
+        const newHeadline = headline (idx, item.webTitle, item.webUrl, item.apiUrl)
+        this._articleHeadline(newHeadline);
+      });
+    });
```

Comment:
This is a nice way to return the element of the response you want and store it in a variable